### PR TITLE
Geocoder impl

### PR DESCRIPTION
**Pull request by mreaugusto**

### CODE REVIEW

**Comment by mreaugusto** [comment]
Hi @amlanc1, I've added more updates to the project. When you have time please take a look and approve the PR if it looks good.

File: rsbc-ride-geocoder-svc/opentelemetry-javaagent.jar
Comment (amlanc1) [inline rsbc-ride-geocoder-svc/opentelemetry-javaagent.jar:1]: Is this file needed to be part of the repo? I see run-service.sh file is downloading the file as a step. Does this jar file need to be committed?

**Review by amlanc1 — CHANGES_REQUESTED** [review]
Update comments inline

**Comment by amlanc1** [comment]
@mreaugusto  Added coments

File: rsbc-ride-geocoder-svc/opentelemetry-javaagent.jar
Comment (mreaugusto) [reply rsbc-ride-geocoder-svc/opentelemetry-javaagent.jar:1]: You're right. It doesn't need to be committed. I'll add it to the .gitignore file but it will be deleted at build time to configure open telemetry to expose metrics to prometheus

Comment (mreaugusto) [reply rsbc-ride-geocoder-svc/opentelemetry-javaagent.jar:1]: Done!
